docker/docker-compose.yml

```diff
@@ -27,8 +39,9 @@ services:
     ports:
       - 8080:8888
     environment:
-      - AWS_ACCESS_KEY_ID
-      - AWS_SECRET_ACCESS_KEY
+      - AWS_ACCESS_KEY_ID=BQA2GEXO711FVBVXDWKM
```

Comment:
Setting the variables here in the docker-compose we are losing the ability to override it dynamically. How about if we just load the variable here -- as it was before -- and the value set will be outside of docker-compose? it applies for 
```
AWS_*
S3_ENDPOINT_URL
```
variables.